docker/test_performance_layers.sh

```diff
@@ -75,5 +75,8 @@ FileCheck "${PROJECT_ROOT_DIR}/test/check_memory_usage_log.txt" \
 FileCheck "${PROJECT_ROOT_DIR}/test/check_frame_time_log.txt" \
   --match-full-lines --input-file "${OUTPUT_DIR}"/frame_time.csv
 
-FileCheck "${PROJECT_ROOT_DIR}/test/check_event_log.txt" \
-  --match-full-lines --input-file "${OUTPUT_DIR}"/events.log
+FileCheck "${PROJECT_ROOT_DIR}/test/check_compile_time_log.txt" \
```

Comment:
This shouldn't be changed I think, we already have compile_time check at the top

Reply:
Sorry, my bad.